[2025-08-18 13:22:56,300] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.verify_pipeline scheduled__2025-08-18T11:22:45.167037+00:00 [queued]>
[2025-08-18 13:22:56,316] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.verify_pipeline scheduled__2025-08-18T11:22:45.167037+00:00 [queued]>
[2025-08-18 13:22:56,316] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-18 13:22:56,316] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2025-08-18 13:22:56,316] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-18 13:22:56,336] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): verify_pipeline> on 2025-08-18 11:22:45.167037+00:00
[2025-08-18 13:22:56,341] {standard_task_runner.py:52} INFO - Started process 4535 to run task
[2025-08-18 13:22:56,345] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'send_weather_to_kafka', 'verify_pipeline', 'scheduled__2025-08-18T11:22:45.167037+00:00', '--job-id', '287', '--raw', '--subdir', 'DAGS_FOLDER/dag_meteo.py', '--cfg-path', '/tmp/tmpfe5m2kxw', '--error-file', '/tmp/tmpx5b8h9d6']
[2025-08-18 13:22:56,347] {standard_task_runner.py:80} INFO - Job 287: Subtask verify_pipeline
[2025-08-18 13:22:56,381] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-18 13:22:56,438] {task_command.py:369} INFO - Running <TaskInstance: send_weather_to_kafka.verify_pipeline scheduled__2025-08-18T11:22:45.167037+00:00 [running]> on host a4b544519f2b
[2025-08-18 13:22:56,552] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=send_weather_to_kafka
AIRFLOW_CTX_TASK_ID=verify_pipeline
AIRFLOW_CTX_EXECUTION_DATE=2025-08-18T11:22:45.167037+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-08-18T11:22:45.167037+00:00
[2025-08-18 13:22:56,553] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-08-18 13:22:56,554] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            echo " Attente traitement Logstash..." && sleep 30 &&\n            curl -s "http://elasticsearch:9200/_cat/indices/weather*" || echo "Pas encore d\'index weather"\n        ']
[2025-08-18 13:22:56,572] {subprocess.py:85} INFO - Output:
[2025-08-18 13:22:56,575] {subprocess.py:92} INFO -  Attente traitement Logstash...
[2025-08-18 13:23:26,624] {subprocess.py:92} INFO - yellow open weather_data BQjbDtS5TvO1Q2HrAJGyvA 1 1 70 0 46.3kb 46.3kb
[2025-08-18 13:23:26,627] {subprocess.py:96} INFO - Command exited with return code 0
[2025-08-18 13:23:26,684] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=send_weather_to_kafka, task_id=verify_pipeline, execution_date=20250818T112245, start_date=20250818T132256, end_date=20250818T132326
[2025-08-18 13:23:26,721] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-18 13:23:26,769] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
